website/docs/cloud-docs/vcs/gitlab-eece.mdx

```diff
@@ -83,10 +83,10 @@ Leave the page open in a browser tab. In the next step you will copy values from
    | Name                    | Terraform Cloud (`<YOUR ORGANIZATION NAME>`)   |
    | Redirect URI            | `https://app.terraform.io/<YOUR CALLBACK URL>` |
    | Confidential (checkbox) | ✔️ (enabled)                                   |
-   | Expire access tokens (checkbox) | ✘ (disabled)                           |
+   | Expire access tokens (checkbox) | (no longer required)                   |
    | Scopes (all checkboxes) | api                                            |
 
-   -> **Note:** Terraform Cloud refreshes tokens if they are older than 6 hours, but GitLab marks OAuth tokens as expired 2 hours after they are issued. Disabling the `Expire access tokens` checkbox prevents Terraform Cloud from attempting to use expired tokens. 
+   -> **Note:** Previous versions of Terraform Cloud required disabling `Expire access tokens` because Gitlab auth tokens would only be refreshed after 6 hours, but Gitlab would expire them after 2 hours. The setting does not exist on newer Gitlab versions.
```

Comment:
```suggestion
   -> **Note:** For previous versions of Terraform Cloud and GitLab, we recommended disabling a setting called `Expire access tokens`. This action was required because Gitlab marked OAuth tokens as expired after 2 hours, but Terraform Cloud only refreshed tokens after 6 hours. This setting does not exist on Gitlab v15+ and Terraform Cloud now refreshes tokens more often.
```

Reply:
This is great, thanks for the suggestion. 😄